### ISSUE TITLE
Unable to start an agent. cli.py  errors out at cli.py , FileNotFoundError: [Errno 2] No such file or directory: './setup'

### ISSUE THREAD

**Issue by swatakit** [Significant-Gravitas/AutoGPT#6675]
### ⚠️ Search for existing issues first ⚠️

- [x] I have searched the existing issues, and there is no existing issue for my problem

### Which Operating System are you using?

Windows

### Which version of AutoGPT are you using?

Latest Release

### Do you use OpenAI GPT-3 or GPT-4?

GPT-3.5

### Which area covers your issue best?

Installation and setup

### Describe your issue.

Hi, I have setup autogpt on windows 11, and have successfully `./run setup`

- githib login and token setup
- able to create agent
- able to add agent to arena

but when i try to start agent, I got file not found error from cli.py --> looking for `./setup`

the repo was forked and cloned 6 Jan, I don't see setup but I only see setup.sh and has already convert dos2unix on it

Can someone help me please
![setup](https://github.com/Significant-Gravitas/AutoGPT/assets/33592900/d6edcbee-8772-4728-9a66-80eee01f1002)

**Comment by ChestnutLons** [comment]
I have the same situation.

**Comment by Kheiden** [comment]
Make sure to run `dos2unix autogpts/<agent name>/setup` when your `pwd` is at the root of the installation.

**Comment by MrCredible** [comment]
> Make sure to run `dos2unix autogpts/<agent name>/setup` when your `pwd` is at the root of the installation.
> 
> You will also need to run `dos2unix autogpts/<agent name>/run_benchmark` and `dos2unix autogpts/<agent name>/run`

This will need to be added into the quickstart guide [here](https://github.com/Significant-Gravitas/AutoGPT/blob/master/QUICKSTART.md#running-your-agent)

**Comment by zenetio** [comment]
I was getting the same issue, but I fixed it with the following changes.
In cli.py, I changed the 3 command lines that are using the subprocess.Popen() like
`setup_process = subprocess.Popen(["./setup"], cwd=agent_dir)`
by the following
`setup_process = subprocess.Popen(["bash ./setup"], shell=True, cwd=agent_dir)`
This tells subprocess.Popen to run the command bash /path/to/setup in a shell, which should execute your setup script.
Now it is working fine.
